cloud/pkg/cloudhub/common/helper_test.go

```diff
@@ -0,0 +1,269 @@
+/*
+Copyright 2022 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"testing"
+
+	"github.com/kubeedge/kubeedge/common/constants"
+	"encoding/json"
+
+	beehivemodel "github.com/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/common/model"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	edgecon "github.com/kubeedge/kubeedge/cloud/pkg/edgecontroller/constants"
+	// "k8s.io/apimachinery/pkg/runtime"
+	// "k8s.io/apimachinery/pkg/runtime/schema"
```

Comment:
Please optimize the grouping of imported packages, and commented codes can be considered for deletion.

Reply:
oh noo, really sorry. fixing it